.github/ISSUE_TEMPLATE/10_bug_report.yml

```diff
@@ -0,0 +1,74 @@
+name: 🐞 Bug Report
+description: Create a report about something that is not working
+body:
+- type: markdown
+  attributes:
+    value: |
+      More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies
+
+      Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting **non-security** bugs and feature requests.
+
+      If you believe you have an issue that affects the SECURITY of the platform, please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) but ONLY if it is reported through email.
+      For other types of questions, consider using [StackOverflow](https://stackoverflow.com).
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: A clear and concise description of what the bug is.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A clear and concise description of what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: |
+      Steps to reproduce the behavior.
+      
+      We ❤ code! Point us to a minimalistic repro project hosted in a public GitHub repo.
+      For a repro project, create a new ASP.NET Core project using the template of your choice, apply the minimum required code to result in the issue you're observing.
+
+      We will close this issue if:
+      - The repro project you share with us is complex. We can't investigate custom projects, so don't point us to such, please.
+      - If we will not be able to repro the behavior you're reporting.
+      - If the repro project is attached as a `.zip` file.
+      - If the GitHub repro project is set to `Private`.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Exceptions (if any)
+    description: Include the exception you get when facing this issue.
+    placeholder: 
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: .NET Version 
+    description: |
+      Run `dotnet --version`
```

Comment:
Why dotnet --version vs dotnet --info?

Reply:
I have `dotnet --info` in the next field. I could just remove this `input` field, but I had added it as `dotnet --version` is usually the most pertinent information for us. 

In the current template we're able to put `dotnet --info` within a `<details>` section so it doesn't take a bunch of vertical space in the issue (with the full `dotnet --info` output you have to scroll a decent amount to get to the comments). With this new template `<details>` doesn't appear to be possible. So we may get a lot of issues that have 100-200 lines of `dotnet --info` related output.